src/components/paradox/results/personality/career/modal/jobs.js

```diff
@@ -1,13 +1,40 @@
 import PropTypes from "prop-types";
+import {
+  faBuilding,
+  faLocationDot,
+  faSuitcase
+} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
 import style from "./style.scss";
 
 function Jobs({jobs}) {
   return (
     <div className={style.list}>
       {jobs.map((job) => (
         <div className={style.listItem} key={job.id}>
-          <div className={style.title}>{job.title}</div>
-          <div className={style.description}>{job.description}</div>
+          <div className={style.job}>
+            <div className={style.jobDetails}>
+              <Icon className={style.jobTitleIcon} icon={faSuitcase} />
+              <div>
+                <div className={style.title}>
+                  {job.title}
+                </div>
+                <div className={style.description}>
+                  <Icon className={style.jobIcon} icon={faBuilding} />
+                  {job.company}
+                </div>
+                <div className={style.description}>
+                  <Icon className={style.jobIcon} icon={faLocationDot} />
+                  {job.location}
+                </div>
+              </div>
+            </div>
+            <div>
+              <a className={style.applyNowButton} href={job.url}>
+                Apply Now
```

Comment:
I think this needs to be translated

Reply:
Ok Translated! Also going to add in some other missing translations later!